htlcswitch/link.go

```diff
@@ -380,6 +380,10 @@ type channelLink struct {
 	// our next CommitSig.
 	incomingCommitHooks hookMap
 
+	// quiescer is the state machine that tracks where this channel is with
+	// respect to the quiescence protocol.
+	quiescer
```

Comment:
It'd be cleaner if we could avoid embedding structs as much as possible.

Reply:
Would you prefer i do something like `quiescer quiescer`?